src/pages/Login/Login.page.jsx

```diff
@@ -1,38 +1,154 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
 import { useHistory } from 'react-router';
 
-import { useAuth } from '../../providers/Auth';
-import './Login.styles.css';
+import styled from 'styled-components';
+import { useAuth } from '../../providers/Auth/Auth.provider';
+
+const Button = styled.button`
+  width: 5rem;
+  margin-top: 1rem;
+  padding: 0.4rem 0.6rem;
+  font-size: 1.2rem;
+  color: ${(props) => props.theme.textcolor};
+  border: none;
+  background-color: transparent;
+`;
+
+const Input = styled.input`
+  color: black;
+  font-size: 1rem;
+  width: 100%;
+  padding: 0.4rem 0.6rem;
+  border-radius: 3px;
+  border: 1px solid white;
+  background-color: ${(props) => props.theme.variant};
+  font-family: Nunito;
```

Comment:
Please make sure to embed this font in your project: https://fonts.google.com/specimen/Nunito?query=nunito

Reply:
removed is not used, thanks